### PR TITLE
Update readme to use readthedocs and clean up

### PR DESCRIPTION
**Pull request by joshuaboniface**
1. Replace all Wiki references to Readthedocs.io references
2. Clean up the layout a bit, remove some redundant bits and make
"about" bits consistent between Docs and README

### CODE REVIEW

**Comment by anthonylavado** [comment]
I didn’t realize my comments didn’t lnd. They’re mostly stale, and I concede on the README contents, so I’ll self dismiss it.

There is one typo however

File: README.md [hunk 1]
```diff
@@ -1,53 +1,37 @@
...
-
-Jellyfin is descended from Emby 3.5.2, ported to the .NET Core framework, and aims to contain build facilities for every platform.
+---
 
-## Issues and Feature Requests
+Jellyfin is a Free Software Media System that puts you in control of managing and streaming your media. It is an alternative to the proprietary Emby and Plex, to provide media from a dedicated server to end-user devices. Jellyfin is descended from Emby's 3.5.2 release and ported to the .NET Core framework to enable full cross-platform support. It aims to contain a server and client applications for every platform, to let you decide where and how you access your media on any of your devices. There are no strings attached, no premium licenses or features, and no hidden agendas: just a team who want to build something better and work together to acheive it. We welcome anyone who is interested in joining us in our quest!
```
Comment (anthonylavado) [inline README.md]: Typo in “achieve”.

File: README.md [hunk 2]
```diff
@@ -1,53 +1,36 @@
...
```
[the remark this answers is not in the document]
Comment (joshuaboniface) [reply README.md]: You know I do agree with you here re-reading it after a good sleep. This sentence doesn't really say anything useful:

>It aims to contain a server and client applications for every platform, to let you decide where and how you access your media on any of your devices. 

If we remove that, the whole thing should be nicer to read in paragraph format.

File: README.md [hunk 3]
```diff
@@ -1,53 +1,37 @@
...
 
+For further details, please see [our documentation page](https://jellyfin.readthedocs.io). To receive the latest updates, get help with Jellyfin, and join the community, please visit [one of our communication channels on Matrix/Riot or social media](https://jellyfin.readthedocs.io/en/latest/user-docs/getting-help).
 
+For more information about the project, please see our [about page](https://jellyfin.readthedocs.io/en/latest/about/).
 
 <p align="center">
-  <strong>New idea or improvement?</strong>
-<em>Open a <a href="https://github.com/jellyfin/jellyfin/wiki/Feature-Requests">Feature Request</a>.</em>
+<strong>Want to get started?</strong>
+<em>Choose from <a href="https://jellyfin.readthedocs.io/en/latest/user-docs/installing/">Prebuilt Packages</a> or <a href="https://jellyfin.readthedocs.io/en/latest/user-docs/building/">Build from Source</a>, the see our <a href="https://jellyfin.readthedocs.io/en/latest/user-docs/quick-start/">Quickstart guide</a>.</em>
```
[the remark this answers is not in the document]
Comment (joshuaboniface) [reply README.md]: Typo fixed.

File: README.md (hunk 1 above)
Comment (joshuaboniface) [reply README.md]: Typo fixed.

**Comment by joshuaboniface** [comment]
Comments addressed. Last thing is to get a wide logo bar image in SVG to use, since GitHub doesn't support the "whole-width black bar" hack I did on readthedocs.

**Comment by joshuaboniface** [comment]
Just needs the banner icon from https://github.com/jellyfin/jellyfin-ux#7

**Comment by anthonylavado** [comment]
Approving, but holding off on merge until you get the banner in. Maybe I’ll get it in if I can.

**Pull request merged**